questions.md

```diff
@@ -0,0 +1,4 @@
+# This is Module 3.1 newbranch readme
+
+* How to select certain team members to review my pull request?
+* How to request reviews from others via email address?
```

Comment:
In order to formally add a reviewer you will need to add them as a collaborator on a repository. Once the review is requested, generally they are notified through email. Currently there is no way to request a review using an email.